shell/platform/fuchsia/flutter/component.cc

```diff
@@ -526,7 +528,7 @@ std::unique_ptr<fml::Mapping> CreateWithContentsOfFile(int namespace_fd,
   return source->GetMapping() == nullptr ? nullptr : std::move(source);
 }
 
-void Application::AttemptVMLaunchWithCurrentSettings(
+/*void Application::AttemptVMLaunchWithCurrentSettings(
```

Comment:
If this function is not actually needed (always lazily init VM?) then please remove it instead of commenting it out.

Reply:
Done in latest CL